README.md

```diff
@@ -149,6 +149,32 @@ GET http://0.0.0.0:5000/recommendations/1?type=UP_SELL
 ]
 ```
 
+### Stateful Action - Like a Recommendation
+When a recommendation is created, like count is default to 0  
+To like an existing recommendation, call   
+```PUT http://0.0.0.0:5000/recommendations/1/recommended-products/2/like```   
+```
+{
+    "likes": 1,
+    "product_id": 1,
+    "recommendation_product_id": 2,
+    "relationship": "UP_SELL"
+}
```

Comment:
This looks like duplicate code from like implementation and thus have conflicts with main branch. Can you run a ```git pull origin main --rebase``` and ```git push origin feature/implement-dislike```?